Controller/Index/Notification.php

```diff
@@ -11,6 +11,7 @@
 
 use Magento\Framework\App\ResponseInterface;
 use SDM\Altapay\Controller\Index;
+use \Magento\Framework\Exception\NotFoundException;
```

Comment:
It appears to be a bit inconsistent if namespaces are prefixed with \ or not. Most style guides says not to do so so I would suggest doing the same. I would also suggest to order the import by name so it's easier to see what packages are being required.